src/__tests__/validate-selector.test.ts

```diff
@@ -29,6 +29,17 @@ describe("validateSelector", () => {
     expect(validate("@SUPPORTS (-webkit-appearance:none) {}")).toBe(false);
   });
 
+  it("@ command is present, but is @media is valid", () => {
```

Comment:
Could we rewrite the new test titles to read more natural like the others? For example:

```suggestion
  it("is valid when the only at-rule present is @media", () => {
```